render.ts

```diff
@@ -11,13 +11,30 @@ import renderToString from 'https://esm.sh/preact-render-to-string@6.1.0?deps=pr
 import { VNode } from 'https://esm.sh/preact@10.17.1?target=es2022'
 import { Context } from './mod.ts'
 
+const HEAD_TAG = '<head>'
+
 export function render(c: Context, Component: VNode) {
   const htmlString = renderToString(Component)
 
   try {
     const { html, css } = extract(htmlString)
-
-    c.res.body = `<style>${css}</style><body>${html}</body>`
+    console.log(css.length)
```

Comment:
remove this please, or only print this when `c.dev` is true

Reply:
oh, I didn't even know that was there still... sorry about that - I was wondering, why I was getting the length in the test logs.

Reply:
all good 😄